codis/pkg/topom/config.go

```diff
@@ -25,8 +25,10 @@ const DefaultConfig = `
 # Set Coordinator, only accept "zookeeper" & "etcd" & "filesystem".
 # for zookeeper/etcd, coorinator_auth accept "user:password" 
 # Quick Start
-coordinator_name = "filesystem"
```

Comment:
Do not modify the default parameter values

Reply:
can you only pull what is needed?